bin/commands/decryptSecrets.js

```diff
@@ -28,8 +28,16 @@ const
   readlineSync = require('readline-sync'),
   ColorOutput = require('./colorOutput');
 
-function commandDecryptSecrets (encryptedSecretsFile, options) {
-  const cout = new ColorOutput(options);
+function commandDecryptSecrets (file, options) {
+  const
+    vault = new Vault(),
+    encryptedSecretsFile = file || process.env.KUZZLE_SECRETS_FILE || vault.defaultEncryptedSecretsFile,
+    cout = new ColorOutput(options);
+
+  if (!encryptedSecretsFile) {
```

Comment:
This test is always false, since `Vault.defaultEncryptedSecretsFile`, being a default value, is guaranteed to always hold a value.